### PR TITLE
Replaced .format with f- strings

### PR DESCRIPTION
**Pull request by leandermaben**
xref #29547
for
- [ ] pandas/util/_print_versions.py
- [ ] pandas/tests/tseries/frequencies/test_to_offset.py

### CODE REVIEW

File: pandas/util/_print_versions.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def get_sys_info() -> List[Tuple[str, Optional[Union[str, int]]]]:
                 ("python-bits", struct.calcsize("P") * 8),
                 ("OS", f"{sysname}"),
                 ("OS-release", f"{release}"),
-                # ("Version", "{version}".format(version=version)),
+                # ("Version", f"{version}"),
```
Comment (jbrockmendel) [inline pandas/util/_print_versions.py]: can you add a comment above "# FIXME: dont leave commented-out"

Comment (leandermaben) [reply pandas/util/_print_versions.py]: Sure

**Comment by alimcmaster1** [comment]
Please can you use 1 PR and push the commit to the same branch? https://github.com/pandas-dev/pandas/pull/31627

Thanks

**Comment by leandermaben** [comment]
#31627 is closed now
There is only one PR

**Comment by jreback** [comment]
thanks @leandermaben

**Comment by leandermaben** [comment]
Thank you so much @jreback

**Pull request merged**